front/src/components/FilmListItem/index.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import CinemaSeanceInfo from './cinemaSeanceInfo'
+import image from '../../images/films/1.jpg'
+import './filmItem.scss'
+
+const FilmItem = ({
+    itemInfo,
+    cityFilter = "Минск" 
+}) => {
+
+    return(
+        <article className="film-item">
+            <section className="film-item__image-container">
+                <img src={image} alt="" className="film-item__image"/>
+            </section>
+            <section className="film-item__description">
+                <h1>{itemInfo.title}</h1>
+                <h3>{itemInfo.date}</h3>
+                <p>{itemInfo.text} </p>
+                <section className="film-item__timetable">
+                    {
+                        itemInfo.cities.find(city => city.name === cityFilter).cinemas.map(itemInfo => 
+                        <CinemaSeanceInfo filmInfo={itemInfo}/>                            
```

Comment:
One indend should be always added for content. Also such filtering is worth to add outside of jsx markup. Also I **strongly** recommend to use followin style for chaining:

```
    itemInfo
        .cities
        .find(city => city.name === cityFilter)
        .cinemas
        .map(itemInfo => 
            <CinemaSeanceInfo filmInfo={itemInfo}/>  
        )
```
The reason is for better reading. My recommendations with explanations for code style here https://github.com/AndreiShenets/code-styles